### PR TITLE
surahTitle component for #15

### PR DESCRIPTION
**Pull request by thabti**
screenshot: screen shot 2017-03-11 at 9 48 33 pm

usage: 

```js
<SurahTitle chapterNumber={1} color="green" height="600px" width="600px" />
```

### CODE REVIEW

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -8,6 +8,8 @@
 const package = require('../package.json');
 
 module.exports = {
+  debug : true,
+  devtool : 'cheap-module-source-map',
```
Comment (thabti) [inline .storybook/webpack.config.js:12]: chrome console will show es6 rather than the compiled code.

File: src/Drawer/index.stories.js [hunk 2]
```diff
@@ -4,12 +4,12 @@ import Drawer from './index';
 
 storiesOf('Drawer', module)
   .add('default', () => (
-    <Drawer>
+    <Drawer open={false}>
```
Comment (thabti) [inline src/Drawer/index.stories.js:7]: react was complaining about missing required prop.

File: utils/generateSvgComponents.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/*eslint-disable */
+const fs = require('fs');
```
Comment (thabti) [inline utils/generateSvgComponents.js]: this is a hack, reads the svg files and turns all surahTitle into react svg components. Not proud of this, we only have to do it once.

File: utils/generateSvgComponents.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+fs.readdir(svgFolder, (err, files) => {
+  if (err) return console.log(err);
+
+  files.map((file, index) => {
+    const svgPath = svgFolder + file;
+    fs.readFile(svgPath, 'utf8', function (err, data) {
+      if (err) return console.log(err);
+      const surahID = `Surah${index}`;
+      const template = () => {
+        const replaceFill = data.replace(/fill\=\"\#*\S{6}\"/g, "fill={props.color}")
```
Comment (thabti) [inline utils/generateSvgComponents.js]: add props for colored surahTitle, remove unsupported svg attributes.

File: .storybook/webpack.config.js (hunk 1 above)
Comment (mmahalwy) [reply .storybook/webpack.config.js:12]: oooo :)

File: src/Drawer/index.stories.js (hunk 2 above)
Comment (mmahalwy) [reply src/Drawer/index.stories.js:7]: +1

**Review by mmahalwy — APPROVED** [review]

**Pull request merged**